forest/cli/src/cli/db_cmd.rs

```diff
@@ -20,15 +20,15 @@ pub enum DBCommands {
 }
 
 impl DBCommands {
-    pub async fn run(&self, config: &Config) {
+    pub async fn run(&self, config: &Config) -> anyhow::Result<()> {
         match self {
             Self::Stats => {
                 use human_repr::HumanCount;
 
                 let dir = db_path(config);
                 println!("Database path: {}", dir.display());
                 let size = fs_extra::dir::get_size(dir).unwrap_or_default();
-                println!("Database size: {}", size.human_count_bytes());
+                Ok(println!("Database size: {}", size.human_count_bytes()))
```

Comment:
It's a coincidence that `println!` happens to return the same type as us. Let's separate these lines to make it clear that we're not actually returning something from `println!`.